lib/ui/startup/startup.dart

```diff
@@ -38,6 +43,16 @@ class Startup extends StatelessWidget {
   }
 
   Widget _homeBuilder({BuildContext context, StartupViewModel viewModel}) {
-    return _home;
+   final repositoryProvider = AppConfig.of(context).repositoryProvider;
+
+   final articleProvider = ArticleListProvider(
+        title: "title",
+        repository: repositoryProvider.getArticleRepository(),
+        group: ArticleCollectionGroup.discovery,
+        relatedTitle: "related",
+      );
+
+    return Provider<ViewModelProviderInterface<ArticleListViewModel>>.value(
```

Comment:
we now inject the articleListProvider here using the Provider lib, and now anything below in the widget tree can get it.

Reply:
yes for sure that would be a good idea 👍